HenriqueEyer/script.js

```diff
@@ -0,0 +1,204 @@
+window.onload = function() {
+reloadList();
+addClickBtnInput(returnById('btn-input'));
+addClickBtnExcluir(returnById('btn-excluir-ALL'));
+addClickBtnExcluirItem(returnById('btn-excluir-item'));
+addClickBtnExcluirCompletos(returnById('btn-excluir-completos'));
+addClickBtnSave(returnById('btn-salvar'));
+addClickBtnDown(returnById('btn-down'));
+addClickBtnUP(returnById('btn-up'));
+
+
+function addClickBtnUP(element){
+    element.addEventListener('click', function(){
+        moveUp(returnById('select'));
+    })
+}
+
+function addClickBtnDown(element){
+    element.addEventListener('click', function(){
+        moveDown(returnById('select'));
+    })
+}
+
+
+
+function moveUp(element){
+    if(element){
+        let elementFather = element.parentNode;
+        elementFather.insertBefore(element, element.previousElementSibling);
+    }
+}
+
+function moveDown(element){
+    if(element){
+        let elementFather = element.parentNode;
+        if( element == elementFather.lastChild ){
+            elementFather.insertBefore( element, elementFather.firstElementChild ); 
+        }else{
+            elementFather.insertBefore(element.nextElementSibling, element ); 
+        }
+    }
+}
+
+
+
+function saveList(){
+    localStorage.clear();
+    let list = returnByTagName('li') ;
+    let position;
+    for ( position = 0; position < list.length; position++ ){
+        if( list[position].textContent != "" ){
+            localStorage.setItem('list'+ position, list[position].textContent);
+        }
+        if ( list[position].className == "completed" ) {
+            localStorage.setItem('completed' + position, 1);
+        } else {
+            localStorage.setItem('completed' + position, 0);
+        }
+    }
+}
+
+function reloadList(){
+    let position;
+    for( position = 0; position < localStorage.length/2 ; position++ ){
+        let text=localStorage[ "list" + position] ;
```

Comment:
O igual deve ter um espaço antes e um depois
```suggestion
        let text = localStorage[ "list" + position] ;
```